packages/core/src/app/checkout/components/BillingStep.tsx

```diff
@@ -0,0 +1,60 @@
+import { type Address } from '@bigcommerce/checkout-sdk/essential';
+import React, { lazy } from 'react';
+
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+import { AddressFormSkeleton, LazyContainer } from '@bigcommerce/checkout/ui';
+
+import { type BillingProps, StaticBillingAddress } from '../../billing';
+import { retry } from '../../common/utility';
+import CheckoutStep from '../CheckoutStep';
+import type CheckoutStepStatus from '../CheckoutStepStatus';
+import type CheckoutStepType from '../CheckoutStepType';
+
+const Billing = lazy(() =>
+    retry(
+        () =>
+            import(
+                /* webpackChunkName: "billing" */
+                '../../billing/Billing'
+            ),
+    ),
+);
+
+export interface BillingStepProps extends BillingProps{
```

Comment:
Shall we extend or just add the props separately? Keep interfaces separate that way.